frontend/src/app/[locale]/search/error.tsx

```diff
@@ -19,7 +19,7 @@ import ServerErrorAlert from "src/components/ServerErrorAlert";
 
 export interface ParsedError {
   message: string;
-  searchInputs: ServerSideSearchParams;
+  searchInputs: OptionalStringDict;
```

Comment:
just a rename to reflect that this type is generic rather than specific in any way